src/main/resources/application.properties

```diff
@@ -1,3 +1,3 @@
-azure.documentdb.uri=put-your-documentdb-uri-here
-azure.documentdb.key=put-your-documentdb-key-here
-azure.documentdb.database=put-your-documentdb-databasename-here
+azure.documentdb.uri=@env.DOCUMENTDB_URI@
+azure.documentdb.key=@env.DOCUMENTDB_KEY@
+azure.documentdb.database=@env.DOCUMENTDB_DBNAME@

```

Comment:
Since we're using `spring-boot-starter-parent` as parent pom, the form `${SOMEVAR}` has no effect but only `@SOMEVAR@` can be filtered. 
See (https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-1.3-Release-Notes#maven-resources-filtering)

Reply:
okay, as long as the environment variables work when we run Maven goals

Reply:
@LGDoor, do not forget to explain in README that Cosmos DB secrets must be specified through environment variables

Reply:
I've already added some description in README [here](https://github.com/Microsoft/todo-app-java-on-azure/pull/19/commits/e52b82914a64cec107b9debe439b93186f36c0b5)